app/src/main/java/com/myfitnesstracker/dto/Nutrition.kt

```diff
@@ -0,0 +1,5 @@
+package com.myfitnesstracker.dto
+
+//Added Data Class to handle live data with all the variables I seen from the unit test
+data class Nutrition (var calories: String, var carbs: String, var protein: String, var fat: String, var name: String) {
```

Comment:
Created data class for the lines from the unit tests:
mvm.fetchNutritionInfo("pho")

if (it.calories == "268" && it.carbs == "41g" && it.protein == "16g" && it.fat == "3g")
I created the data class based off these two lines

Reply:
Looks good.

Are each of these attributes required?  If not, consider giving some of them default values, so that you only need to pass in required parameters to construct an object, and optional parameters are optional.  A little Kotlin shortcut.